modules/ROOT/pages/storage.adoc

```diff
@@ -0,0 +1,57 @@
+= Configuring Storage
+
+Fedora bootc is a container image that can be "materialized" into a disk image in 3 different ways:
+
+- Anaconda (kickstart)
+- bootc-image-builder
+- `bootc install`
+
+== Included storage technologies
+
+The `full` images include support for:
+
+- `xfs`/`ext4`
+- `btrfs` (Fedora only)
+- LVM
+- LUKS
+
+But you can of course add other storage packages directly to the host system.
+
+=== Storage with Anaconda
+
+You can use https://docs.fedoraproject.org/en-US/fedora/f36/install-guide/advanced/Kickstart_Installations/[Kickstart]
+to configure storage with Anaconda.
+
+=== Storage with bootc-image-builder
+
+You can use the https://github.com/osbuild/bootc-image-builder[bootc-image-builder] bootc-image-builder` tool to create a disk image; in the future more storage configuration will be supported
+
+=== Storage with `bootc install`
```

Comment:
I think this is a good example where I just think things are too much.. 

![Screenshot 2024-04-23 at 12-03-39 Configuring Storage](https://github.com/dustymabe/fedora-coreos-docs/assets/4530030/fd102b45-2b9b-41f4-8155-ca1b09147ed5)


There are too many options.

Reply:
Yeah, one eventual goal is that we make anaconda a container image - then it can be executed in multiple places, including in clouds.  But it's also got giant established workflows where it is the "outer" thing.  So that led to this situation.

Reply:
i would love this!

I would love for wherever you are you start with a Fedora CoreOS image (known working, passed CI, works if you want to just use that, or if you want to change to something derived) as the starting point for everything else.

1. FCOS as your $HOME - you run containers, don't modify heavily, get auto updates on streams
2. FCOS as your installer -> switch to your derived thing, full customization of the disk partitioning, etc..
    - This could be bootc doing the disk partitioning or anaconda (both in a container image)
4. OR the user uses OSBuild/image builder to build their own disk image without the first hop